utils/tps/src/main.rs

```diff
@@ -44,7 +161,7 @@ pub async fn calc_tps(api: &Api, n: usize, genesis: bool) -> Result<(), Error> {
 				.unwrap();
 			(2, block_timestamp_genesis)
 		},
-		false => {
+		true => {
```

Comment:
why were these booleans flipped?

the original rationale was: if `--genesis` is present as a CLI arg (which implies `genesis == true`), when we start scraping from genesis

now, we're explicitly calling `--genesis` when we **don't** want to scrape from genesis?

Reply:
Yep typo, my bad! Fixed it back now.